crates/forge/tests/integration/gas.rs

```diff
@@ -32,11 +36,20 @@ fn declare_cost_is_omitted() {
 
     assert_passed(&result);
     // 1 = cost of 230 steps (because int(0.0025 * 230) = 1)
-    assert_gas(&result, "declare_cost_is_omitted", 1);
+    // 0 l1_gas + 0 l1_data_gas + 1 * (100 / 0.0025) l2 gas
```

Comment:
So the calculation here is now that first it computes the amount of L1 gas (1 because of 230 steps) and then it converts to L2?

Reply:
no, l1_gas is totally different now, and it does not cover execution costs (afaik it just handles messaging).
the calculation here is: there was 1 execution cost from steps, and we jsut convert it to L2 gas (because l2 gas is now a measure of execution cost)

Reply:
Right, but this cost of `1` is measured from 230 steps which are currently priced in l1 gas? 
<img width="1016" alt="image" src="https://github.com/user-attachments/assets/ccb0675c-5475-4b4d-98be-edb6d92b5c7d" />

Reply:
It's:

1 cairo step = 0.0025 L1 gas = 100 L2 gas

230 steps * 0.0025 l1 = 0.575, so 1 (because this is the smallest permissible value)
230 steps * 100 l2 = 23000 (but the same logic as above applies, so minimal value becomes 40000, since 1 l1 gas = 40000 l2 gas)

Reply:
Can we extend the comment so it explains what you just wrote? It wasn't trivial for me.

Reply:
I mean, all this is already in the top comment (with linked docs on calculations), I can add this here for one test as well, but I don't think adding it everywhere will bring any value tbh. 🤔 

Done in newest.